pack/core/core.json

```diff
@@ -240,6 +240,7 @@
         "faction_code": "neutral",
         "flavor": "I ain't going to the pen a second time.",
         "illustrator": "John Pacer",
+        "is_unique": true,
```

Comment:
@Shawnecy this is the `is_unique` that should be removed before we can merge this.